.github/workflows/lint-and-codecov.yml

```diff
@@ -0,0 +1,41 @@
+name: Lint & Generate CodeCov Report
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  coverage:
+    name: Lint & Generate Coverage Report
+    runs-on: ubuntu-20.04
+    container:
+      image: public.ecr.aws/m1r4u2a2/core:latest 
```

Comment:
shall we switch to the new friendly name?

Reply:
Good call. I'll replace this.